include_core/omrgcconsts.h

```diff
@@ -132,6 +132,11 @@ typedef enum MM_ScavengeScanReason {
 #define OMR_GC_CYCLE_TYPE_GLOBAL      1
 #define OMR_GC_CYCLE_TYPE_SCAVENGE    2
 #define OMR_GC_CYCLE_TYPE_EPSILON	 6
+/**
+ *  the state for indicating the cycle is abort, it only is used in the cycleType of CycleEnd event.
+ *  MM_CycleState->_type has uintptr_t type. This code still need to support 32-bit JVMs, so bit needs to fit 32 bit word.
```

Comment:
rephrased to improve readability. 
```suggestion
 *  This bit represents the state to abort a cycle. It is only used in the cycleType of CycleEnd event.
 *  MM_CycleState->_type has uintptr_t type. This code still needs to be supported on 32-bit platforms, which will require the bit to fit in a 32 bit word.
```